std_msgs/Header header
uint8[] serialized_sdf
uint8 scalar_type 0
uint8 SCALAR_TYPE_UNKNOWN=0
uint8 SCALAR_TYPE_FLOAT=1
uint8 SCALAR_TYPE_DOUBLE=2
bool is_compressed
